cloudpickle/cloudpickle.py

```diff
@@ -238,7 +238,11 @@ def dump(self, obj):
 
     def save_memoryview(self, obj):
         """Fallback to save_string"""
-        Pickler.save_string(self, str(obj))
+        try:
+            arg = obj.obj
+        except ValueError:
+            arg = _released_memory_view_sentinel
+        self.save_reduce(_fill_memoryview, (arg,), obj=obj)
```

Comment:
Is there any plan to expose the offset of a memoryview at the Python level in CPython 3.7 or later? If so we could only support pickling of memoryviews for the compatible python versions.